multiversion/v1_18_R2/src/main/java/iridiumcore/nms/NMS_V1_18_R2.java

```diff
@@ -42,7 +44,11 @@ public void deleteBlockFast(Location location) {
      */
     @Override
     public void sendChunk(List<Player> players, org.bukkit.Chunk chunk) {
-        chunk.getWorld().refreshChunk(chunk.getX(), chunk.getZ());
+        net.minecraft.world.level.chunk.Chunk c = ((CraftChunk)chunk).getHandle();
```

Comment:
Stay consistent with abbreviations, we don't use them
```suggestion
        net.minecraft.world.level.chunk.Chunk chunk = ((CraftChunk) chunk).getHandle();
```